lib/account/models/user_label.dart

```diff
@@ -0,0 +1,86 @@
+import 'package:drift/drift.dart';
+import 'package:flutter/foundation.dart';
+
+import 'package:thunder/core/database/database.dart';
+import 'package:thunder/main.dart';
+import 'package:thunder/utils/instance.dart';
+
+class UserLabel {

```

Comment:
Add a description of what this class is used for (just for clarity in the future)!

Reply:
Done!